src/pages/Chart/paywallBoundaries.js

```diff
@@ -1,40 +1,59 @@
 import { getTimeIntervalFromToday, DAY, MONTH } from '../../utils/dates'
 
-const options = { isUTC: true }
+function createOptions (hours = 48) {
+  const NEXT_DAY = new Date()
+  NEXT_DAY.setHours(hours, 0, 0, 0)
+  return { isUTC: true, from: NEXT_DAY }
+}
+
+const { from: ANON_LEFT_BOUNDARY_DATE } = getTimeIntervalFromToday(
+  -3,
+  MONTH,
+  createOptions(72)
+)
```

Comment:
Not clear for me...you set here `72` hours? Or what's idea?

Reply:
Maybe make an interface as 
getTimeIntervalFromToday({cutMoreThanLastN: '3m', cutLastN: '2d' or '24h'})